res/controllers/engine-api.d.ts

```diff
@@ -301,4 +301,41 @@ declare namespace engine {
      *               SoftStart with low factors would take a while until sound is audible. [default = 1.0]
      */
     function softStart(deck: number, activate: boolean, factor?: number): void;
+
+    enum WellKnownCharsets {
```

Comment:
```suggestion
    enum Charsets {
```

Reply:
WellKnownCharsets still appears here

Reply:
whoops yeah, I only `grep` in `src/` and forgot to double check. Fixed and amended.